tests/foreman/api/test_host.py

```diff
@@ -1184,6 +1190,44 @@ def test_negative_update_os(self):
         self.assertNotEqual(
             host.read().operatingsystem.read().name, new_os.name)
 
+    @tier1
+    @skip_if_bug_open('bugzilla', 1427192)
+    def test_positive_read_puppet_proxy_name(self):
+        """Read a hostgroup created with puppet proxy and inspect server's
+        response
+
+        @id: 8825462e-f1dc-4054-b7fb-69c2b10722a2
+
+        @Assert: Field 'puppet_proxy_name' is returned
+
+        @BZ: 1371900
```

Comment:
Blocked by BZ 1427192 but addresses BZ 1371900? Both bugs have 6.2.z+, so if the test validates both maybe worth updating docstring with both of them?